utils/check_bad_commit.py

```diff
@@ -47,6 +47,9 @@ def create_script(target_test):
 
 if len(result.stderr) > 0:
     if "ERROR: file or directory not found: " in result.stderr:
```

Comment:
This happens if the test file or directory is new, i.e. not present in a previous commit